python/py-tensorflow/Portfile

```diff
@@ -42,28 +41,31 @@ if { ${os.major} < ${min_darwin} } {
 
 # Required java version
 java.version        11+
-# JDK port to install if required java not found
-java.fallback       openjdk14
+# LTS JDK port to install if required java not found
+java.fallback       openjdk11
 # JDK only needed at build time, but java PG sets lib dependency so
 # declare no conflict to allow redistribution of binaries.
 license_noconflict  ${java.fallback}
 
-# require c++14
 compiler.cxx_standard 2014
 
 # The oldest Xcode version to use default Xcode compiler
 # https://github.com/tensorflow/tensorflow/issues/39262
 set tf_min_xcode    10.2
 compiler.blacklist-append {clang < 1001}
-# Work out if we should be using macports clang
-set use_mp_clang [ expr ( [ string match macports-clang-* ${configure.compiler} ] || [ vercmp ${xcodeversion} ${tf_min_xcode} ] < 0 ) ]
 
 # https://trac.macports.org/ticket/59192
 xcode_workaround.type                  avoid_xcode_compiler
 xcode_workaround.os_versions           18 19
 xcode_workaround.fixed_xcode_version   11.4
 
-#patch.pre_args      -p1
+# Work out if we should be using macports clang
+set use_mp_clang [ expr ( [ string match macports-clang-* ${configure.compiler} ] || [ vercmp ${xcodeversion} ${tf_min_xcode} ] < 0 ) ]
+
+patch.pre_args      -p1
+
+# Remove when updating to 2.4.x
+patchfiles-append   patch-numpy-1.19-fix.diff
```

Comment:
Please add a comment here to the upstream github issue it fixes. I know the links are in the patch files, but I prefer the reference to also be in the portfile.

Reply:
Links added in  10b1dc4952e